int32 start
int16[] pixels
